apps/articles/index.coffee

```diff
@@ -22,3 +22,4 @@ app.get '/articles', routes.articles
 app.get sd.TEAM_BLOGS, routes.teamChannel
 app.get '/venice-biennale-2015', routes.section
 app.post '/editorial-signup/form', routes.editorialForm
+app.get '/gallery-insights/opt-in', (_, res) -> res.redirect '/gallery-insights'
```

Comment:
`/gallery-insights` doesn't currently have a signup, is that fine? Just the individual article pages do right now.